docs/advanced.rst

```diff
@@ -1132,6 +1132,61 @@ A ``system.conf`` section could look like this:
   region-start=1M
   region-size=64M
 
+.. _sec-raw-partition-fallback:
+
+Update Raw Boot Partition with Fallback
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some SoCs (like the Rockchip RK3568) contain a fixed ROM code that searches the
+possible boot media for valid images in a defined order.
+Usually this happens at two or more fixed addresses.
+Special headers and/or magic numbers are typically used to determine if there
+is a valid image at such an address.
```

Comment:
Yes, perhaps. To be honest, I'm not familiar with the exact behavior of many ROM code loaders.
However, since that could likely be the case, I'll mention that as well.